R/provider-cortex.R

```diff
@@ -33,7 +33,8 @@ NULL
 #'   `account`.
 #'
 #' @param account A Snowflake [account identifier](https://docs.snowflake.com/en/user-guide/admin-account-identifier),
-#'   e.g. `"testorg-test_account"`.
+#'   e.g. `"testorg-test_account"`. Defaults to the value of the
+#'   `SNOWFLAKE_ACCOUNT` environment variable.
```

Comment:
Want to cross reference the two snowflake functions?

Reply:
Done.